src/node.cc

```diff
@@ -877,8 +877,7 @@ Local<Value> ErrnoException(Isolate* isolate,
 
   Local<String> path_string;
   if (path != nullptr) {
-    // FIXME(bnoordhuis) It's questionable to interpret the file path as UTF-8.
-    path_string = String::NewFromUtf8(env->isolate(), path);
+    path_string = FIXED_ONE_BYTE_STRING(env->isolate(), path);
```

Comment:
I don't disagree and to be clear, I don't advocate making changes haphazardly, that wouldn't help.

Aside (and for background): node's use of UTF-8 for file paths is accidental, it wasn't a design choice.  V8 originally only supported UTF-8, one-byte strings weren't added until much later.